clients/lcdproc/disk.c

```diff
@@ -124,32 +137,71 @@ disk_screen(int rep, int display, int *flags_ptr)
 	sock_printf(sock, "widget_set D f 1 2 %i %i %i %i v 12\n", lcd_wid, lcd_hgt, lcd_wid, count);
 	for (i = 0; i < count; i++) {
 		char tmp[lcd_wid + 1];	/* should be large enough */
+		char cap[8] = {'\0'};
+		char dev[256];
```

Comment:
Can we use dev_wid here instead of the magic number 256?

Reply:
Addressed in #181